### DISCUSSION TITLE
does this allow jit on ios 17

### DISCUSSION THREAD

**Discussion by dragonplus8** [doronz88/pymobiledevice3#477] [Q&A]
i imagine not but thought i would ask

**Comment by doronz88** [comment]
I couldn't understand your question. Does what? And JIT in what matter?

**Reply by dragonplus8** [reply]
so for the confusion i mean for enabling jit on an app as you seem to have fixed the personal disk image problem

**Reply by doronz88** [reply]
Jit as in dynamic code signing? Only a very few application have this feature.

Currently you can Mount the new developer disk image but the actually implementation of all the developer relays is still a wip
